.env.sample

```diff
@@ -83,3 +83,8 @@ RABBITMQ_PASSWORD=
 # PACS extraction API
 PIXL_DICOM_TRANSFER_TIMEOUT=240
 PIXL_QUERY_TIMEOUT=10
+
+# FTP server
+FTP_HOST=
+FTP_USER_NAME=
+FTP_USER_PASS=
```

Comment:
Thinking about it, probably should be `FTP_USER_PASSWORD` for consistency across all our other envs